nservicebus/azure-service-bus/publisher-names-configuration.md

```diff
@@ -9,17 +9,17 @@ tags:
 reviewed: 2016-04-18
 ---
 
-When `EndpointOrientedTopology` is selected, a mapping between publisher names and message types has to be properly configured, in order to ensure that subscriber receives event messages.  
+When the [`EndpointOrientedTopology`](/nservicebus/azure-service-bus/topologies/#version-7-and-above-endpoint-oriented-topology) is selected, a mapping between publisher names and message types has to be properly configured, in order to ensure that subscriber can receive event messages.  
 
-The mapping can be configured for specific message type:
+The mapping can be configured for a specific message type:
 
 snippet: publisher_names_mapping_by_message_type
 
 
-or for an assembly containing message types:
+or for an assembly containing multiple message types:
 
 snippet: publisher_names_mapping_by_assembly
 
-In this case, the transport analyzes all types in the assembly to identify which are messages, according to the configured [`message conventions`](/nservicebus/messaging/conventions.md). Then for each message type, the transport registers a mapping between the type and the publisher name.
+In the latter case, the transport analyzes all types in the assembly to identify which are messages, according to the configured [`message conventions`](/nservicebus/messaging/conventions.md). Then for each message type, the transport registers a mapping between the type and the publisher name.
```

Comment:
Hm @melkio told me something else, I remember we talked about it specifically. Could you confirm?